app/src/main/java/tech/relaycorp/letro/data/PeerType.kt

```diff
@@ -0,0 +1,6 @@
+package tech.relaycorp.letro.data
+
+sealed interface PeerType {
+    object Public : PeerType
+    object Private : PeerType
+}
```

Comment:
This looks like an enum defined in a strange way. Why not just use an enum?

UPDATE: Is this actually used?

Reply:
Deleted